src/shogun/classifier/svm/SVMSGD.cpp

```diff
@@ -214,13 +214,32 @@ void CSVMSGD::init()
 	loss=new CHingeLoss();
 	SG_REF(loss);
 
-    m_parameters->add(&C1, "C1",  "Cost constant 1.");
-    m_parameters->add(&C2, "C2",  "Cost constant 2.");
-    m_parameters->add(&wscale, "wscale",  "W scale");
-    m_parameters->add(&bscale, "bscale",  "b scale");
-    m_parameters->add(&epochs, "epochs",  "epochs");
-    m_parameters->add(&skip, "skip",  "skip");
-    m_parameters->add(&count, "count",  "count");
-    m_parameters->add(&use_bias, "use_bias",  "Indicates if bias is used.");
-    m_parameters->add(&use_regularized_bias, "use_regularized_bias",  "Indicates if bias is regularized.");
+	m_parameters->add(&C1, "C1", "Cost constant 1.");
+	watch_param("C1", &C1);
+
+	m_parameters->add(&C2, "C2", "Cost constant 2.");
+	watch_param("C2", &C2);
+
+	m_parameters->add(&wscale, "wscale", "W scale");
+	watch_param("wscale", &wscale);
+
+	m_parameters->add(&bscale, "bscale", "b scale");
+	watch_param("bscale", &bscale);
+
+	m_parameters->add(&epochs, "epochs", "epochs");
+	watch_param("epochs", &epochs);
+
+	m_parameters->add(&skip, "skip", "skip");
+	watch_param("skip", &skip);
+
+	m_parameters->add(&count, "count", "count");
+	watch_param("count", &count);
+
+	m_parameters->add(&use_bias, "use_bias", "Indicates if bias is used.");
+	watch_param("use_bias", &use_bias);
```

Comment:
Actually, we just realised that we probably want to keep the comment as well.
But that requires a minor change to the registration method .... Let me cook something up and ping you